src/stylesheets/components/_navigation.scss

```diff
@@ -35,15 +35,14 @@ $navigation-height: 50px;
 
 .app-navigation__link {
   @include govuk-typography-weight-bold;
-  text-decoration: none;
+  @include govuk-link-style-no-underline;
 
   &:not(:focus):visited {
     color: $govuk-link-colour;
   }
```

Comment:
Good spot. I've removed it in a separate commit as this PR didn't touch the visited styles. 

I've also now used classes instead of mixins to set the new link styles to align with how we set the no-visited-state in the markup.